packages/contract-wrappers/src/index.ts

```diff
@@ -1,7 +1,31 @@
 export { ContractAddresses } from '@0x/contract-addresses';
 
 export { ContractWrappers } from './contract_wrappers';
-
+export { DevUtilsContract } from './generated-wrappers/dev_utils';
+export {
+    ERC1155MintableContract,
```

Comment:
Why are we exporting the `Mintable` version of this contract rather than the regular version? We only export the regular versions for ERC20/721.

Reply:
Hm I don't know. This was copied from `abi-gen-wrappers`/ derived from `@0x/contract-artifacts`. In `contract-artifacts` there's only `ERC1155Mintable` and *not* `ERC1155Token`.

Reply:
I think we can just remove `ERC1155MintableContract`. But should we add `ERC1155TokenContract`?

Reply:
Yep, let's remove it and add ERC1155Token